WISE-Community/WISE
# Added team sign in to site (including Google sign in)

To test:
- Log in as a student
- Test out launching runs
- Make sure all teammates can sign in before launching a run (including both regular WISE users and Google WISE users)
- Make sure you can add users to a team (both regular and Google users)
- Make sure marking users as absent works

Closes #1676.
